prisonstaffhub-specs/src/test/groovy/uk/gov/justice/digital/hmpps/prisonstaffhub/mockapis/Elite2Api.groovy

```diff
@@ -950,5 +950,187 @@ class Elite2Api extends WireMockRule {
         )
     }
 
+    void stubAlertTypes(response = []) {
```

Comment:
Added these just to get the integration tests working, but plan is to move them to cypress anyway.